src/services/jsTyping.ts

```diff
@@ -40,16 +40,15 @@ namespace ts.JsTyping {
     export function discoverTypings(
         host: TypingResolutionHost,
         fileNames: string[],
-        cachePath: Path,
         projectRootPath: Path,
         safeListPath: Path,
-        packageNameToTypingLocation: Map<string>,
+        packageNameToTypingLocation: Map<Path>,
         typingOptions: TypingOptions,
         compilerOptions: CompilerOptions):
-        { cachedTypingPaths: string[], newTypingNames: string[], filesToWatch: string[] } {
+        { cachedTypingPaths: Path[], newTypingNames: string[], filesToWatch: string[] } {

```

Comment:
As discussed, I'm no longer using the filesToWatch on the VS host side.  (It is also currently returning package.json paths that don't actually exist).  It looks like you don't need this either.  Can we remove this?